src/Cryptol/ModuleSystem/Name.hs

```diff
@@ -21,7 +21,6 @@
 module Cryptol.ModuleSystem.Name (
     -- * Names
     Name(), NameInfo(..)
-  , NameSource(..)
```

Comment:
We could still export `NameSource(..)` from here, even though it is not defined in this module---it would probably save on changing some imports

Reply:
Let's reexport this from there, so we don't need all the extra imports in the other places.